### ISSUE TITLE
Implement a Benchmark to messure Tomcat performance

### ISSUE THREAD

**Issue by maximilianhuber** [sw360/sw360portal#397]
It would be great to have some number how "fast" the tomcat is.

Maybe that can combined with some potential selenium tests.

**Comment by mcjaeger** [comment]
I think a simple thing could be track test execution times to a log.

**Comment by maximilianhuber** [comment]
related Issue is: https://github.com/sw360/sw360chores/issues/24 "Configure browser caching in nginx"

**Comment by maximilianhuber** [comment]
@mcjaeger no, the tests are executed without the tomcat behind them. Selenium tests would work. (probably).

I would like to see something which measures how long sw360 takes to render the complete Project page data or how long it takes to execute some simple workflow in the UI. The goal would be to have tests which check, that the responsiveness of the UI does not decrease.

**Comment by mcjaeger** [comment]
a first solution could be to have it similar as in fossology, where some query times are delivered as part of the requests.
